internal/xds/translator/translator_test.go

```diff
@@ -163,12 +165,17 @@ func TestTranslateXds(t *testing.T) {
 			routes := tCtx.XdsResources[resourcev3.RouteType]
 			clusters := tCtx.XdsResources[resourcev3.ClusterType]
 			endpoints := tCtx.XdsResources[resourcev3.EndpointType]
+			overrideTestDataOutFile(t, requireResourcesToYAMLString(t, listeners), "xds-ir", tc.name+".listeners.yaml")
```

Comment:
why is this override needed ?

Reply:
Trying to update the tons of json/yaml output files automatically by running `make test`, otherwise I need to manually update all of them manually (40+ files).

Reply:
I love the idea, can we use a seperate `make` target to achieve this, as well as in a follow up PR
running `make test` to check errors and fix errors feels fishy, how about `make testdata` to perform this specific task.

Reply:
Done : )